src/map/homunculus.cpp

```diff
@@ -697,15 +678,7 @@ int hom_mutate(struct homun_data *hd, int homun_id)
 		return 0;
 	}
 
-	hom_calc_skilltree(hd);
```

Comment:
Did you intentionally remove this?

Reply:
Same as above, but in this case I guess I should revert it, since this is partially custom. On official servers, the homunculus is called AFTER mutation. On the official quest you can't call it after you receive the Strange embryo, then at the end of the quest you receive an already mutated homun.
I may need to review this whole function on another PR.